docs/stdlib/bool.rst

```diff
@@ -40,10 +40,9 @@ Booleans
 
 .. eql:type:: std::bool
 
-    A boolean type with possible values of ``true`` and ``false``.
+    Represents a logical boolean type of either ``true`` or ``false``.
```

Comment:
```suggestion
    A boolean type of either ``true`` or ``false``.
```

Since "boolean" implies "logical" and there is no representation happening since this is the type that we're describing.